MultiQC/1.5/MultiQC.nf

```diff
@@ -6,15 +6,17 @@ process MultiQC {
     container = 'library://sawibo/default/bioinf-tools:multiqc-1.5'
 
     input:
-    val id
+
```

Comment:
Would it be an idea to keep id and pass it to -f ?  We could set the title of the report based on the run id for example.

Reply:
Goal was here to make this module similar to the 1.8 version and id is not really needed imo